src/test/javascript/spec/helpers/sample/metis-sample-data.ts

```diff
@@ -0,0 +1,139 @@
+import { Course } from 'app/entities/course.model';
```

Comment:
I really like the extraction of the sample data, since it greatly reduces duplication. This, however, would have been one aspect that could have been split into a second PR to reduce the size of this PR

Reply:
point taken...